src/tiled/tiled.qbs

```diff
@@ -189,6 +189,8 @@ QtGuiApplication {
         "flipmapobjects.h",
         "geometry.cpp",
         "geometry.h",
+        "gotodialog.cpp",
+        "gotodialog.h",
```

Comment:
Please add the new files to the `tiled.pro` file as well. The builds for Linux and macOS packages unfortunately still rely on qmake.